project2/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=ZCOOL+QingKe+HuangYou|Khand" rel="stylesheet">
+</head>
+<title>Card Specimen of Spider-Man</title>
+
+<body>
+    <div class="rapper">
+        <figure>
+            <img src="images/Spiderman.jpg" alt="spider-man with black background" width="200px" height="200px">
+        </figure>
+    </div>
+    <div class="context">
+        <h1 class="title" style="width: 360px;">Spider-Man: Into the Spider Verse</h1>
+        <div class="rate">
+            <span>8.6/10 </span>
+            <span class="length">
+                1h 56m</span>
+            <span class="playbutton">
+                <img src="images/play.png" alt="play button" width="45px" height="45px"></span>
```

Comment:
See previous note about making this a link around the image:
```suggestion
                <img src="images/play.png" alt="play button" width="45px" height="45px"></a>
```